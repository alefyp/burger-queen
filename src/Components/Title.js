import React from 'react';
import styles from '../css/Titles.module.css'

//texto y color

const Title = (props) => {
  return (
    <h2 style={{ color: props.color }} className={styles.title}>{props.text}</h2>
  );
}

export default Title;