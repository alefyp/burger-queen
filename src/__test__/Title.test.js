import React from "react";
import renderer from "react-test-renderer";
import Title from "../Components/Title";

it("Matches Title Component Snapshot", () => {
  const tree = renderer.create(<Title text="Hello world!" />).toJSON();
  expect(tree).toMatchSnapshot();
});
