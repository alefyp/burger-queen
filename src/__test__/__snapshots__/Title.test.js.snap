// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches Title Component Snapshot 1`] = `
<h2
  className="title"
  style={
    Object {
      "color": undefined,
    }
  }
>
  Hello world!
</h2>
`;
